relay/messages/execution.go

```diff
@@ -73,11 +72,11 @@ var errorCommandNotFound = errors.New("Command not found")
 // ToCircuitRequest converts an ExecutionRequest into a circuit.api.ExecRequest
 func (er *ExecutionRequest) ToCircuitRequest(bundle *config.Bundle, relayConfig *config.Config, useDynamicConfig bool) (*api.ExecRequest, bool, error) {
 	retval := &api.ExecRequest{}
-	hasDynamicConfig := er.compileEnvironment(retval, relayConfig, useDynamicConfig)
-	command := bundle.Commands[er.commandName]
+	command := bundle.Commands[er.CommandName()]
 	if command == nil {
 		return nil, false, errorCommandNotFound
 	}
+	hasDynamicConfig := er.compileEnvironment(command, retval, relayConfig, useDynamicConfig)
```

Comment:
Can we rename this variable? It's a little misleading now, seeing as how we're doing more than just dealing with dynamic config. Maybe something like `withEnvironment`, or similar?